sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -217,6 +225,8 @@ class RunInference(beam.PTransform[beam.PCollection[ExampleT],
   Args:
       model_handler: An implementation of ModelHandler.
       clock: A clock implementing get_current_time_in_microseconds.
+      extra_kwargs: Extra arguments for models whose inference call requires
+        extra parameters.
```

Comment:
I think that PythonDocs PreCommit is unhappy with the indentation here (not sure though). You should be able to reproduce the failure locally with `tox -e py38-docs` to test a fix.

Reply:
Thanks. Fixed.